app/bundles/NotificationBundle/Form/Type/ConfigType.php

```diff
@@ -78,6 +78,32 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ]
         );
+        $builder->add(
+            'gcm_sender_id',
+            'text',
+            [
+                'label' => 'mautic.notification.config.form.notification.gcm_sender_id',
+                'data'  => $options['data']['gcm_sender_id'],
+                'attr'  => [
+                    'tooltip'      => 'mautic.notification.config.form.notification.gcm_sender_id.tooltip',
+                    'class'        => 'form-control',
+                    'data-show-on' => '{"config_notificationconfig_notification_enabled_1":"checked"}',
+                ],
+            ]
+        );
+
+        $builder->add(
+            'welcomenotification_enabled',
+            'yesno_button_group',
+            [
+                'label' => 'mautic.notification.config.form.notification.welcome.enabled',
+                'data'  => (bool) $options['data']['welcomeNotificationEnabled'],
```

Comment:
I'm still getting `Notice: Undefined index: welcomeNotificationEnabled` on this line. Shouldn't it be `welcomenotification_enabled` from `app/bundles/NotificationBundle/Config/config.php`? Try to remove it from your local.php to test the problem.